src/Reader.php

```diff
@@ -62,6 +62,14 @@ class Reader extends AbstractCsv implements Countable, IteratorAggregate, JsonSe
     protected $stream_filter_mode = STREAM_FILTER_READ;
 
     /**
+     * @inheritdoc
+     */
+    public static function createFromPath(string $path, string $open_mode = 'r', $context = null): AbstractCsv
```

Comment:
This introduces a tiny BC break you can't just correct it like this

Reply:
Since you'd mentioned this would potentially go in a 10.0 release (which is likely to be atleast a year away), I approached it with that in mind. Is there another way I should approach this PR?

Reply:
The return type-hint here is wrong. You're returning `new static` so the return type should be `Reader`. While upgrading from 9.0.1 to 9.1.2 I started getting errors like these from Phan:

```
app/Legacy/Console/Commands/AbstractCsvImportCommand.php:77 PhanUndeclaredMethod Call to undeclared method \League\Csv\AbstractCsv::setHeaderOffset
```